### PR TITLE
Prepare Core Libraries for September 2021 Release

### PR DESCRIPTION
**Pull request by alzimmermsft**

### CODE REVIEW

File: sdk/core/azure-core-http-okhttp/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
 # Release History
 
-## 1.8.0-beta.1 (Unreleased)
+## 1.7.3 (Unreleased)
```
Comment (srnagar) [inline sdk/core/azure-core-http-okhttp/CHANGELOG.md:3]: Update the dates too in all the changelogs.

File: sdk/core/azure-core-http-okhttp/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
 
+### Features Added
+
+- Added support for additional environment configurations. ([#23435](https://github.com/Azure/azure-sdk-for-java/pull/23435))
+
+### Dependency Updates
```
Comment (srnagar) [inline sdk/core/azure-core-http-okhttp/CHANGELOG.md:9]: I think we now include dependency updates under "Other changes" section.

File: sdk/core/azure-core-http-netty/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
 
-## 1.11.0-beta.1 (Unreleased)
+## 1.11.0 (Unreleased)
 
+### Features Added
+
+- Added the ability to configure HTTP connect timeout. ([#23435](https://github.com/Azure/azure-sdk-for-java/pull/23435))
+- Added support for additional environment configurations. ([#23435](https://github.com/Azure/azure-sdk-for-java/pull/23435))
+
+### Dependency Updates
```
Comment (g2vinay) [inline sdk/core/azure-core-http-netty/CHANGELOG.md:10]: Following latest changelog guidelines, 
we can move this section under `Other Changes` header.
Here and other places too in the PR.

### Other Changes
#### Dependency Updates

**Review by g2vinay — APPROVED** [review]

**Review by kasobol-msft — APPROVED** [review]

File: eng/jacoco-test-coverage/pom.xml [hunk 4]
```diff
@@ -139,17 +139,17 @@
...
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-okhttp</artifactId>
-      <version>1.8.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;current} -->
+      <version>1.7.3</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-management</artifactId>
-      <version>1.5.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-management;current} -->
+      <version>1.4.1</version> <!-- {x-version-update;com.azure:azure-core-management;current} -->
```
Comment (pallavit) [inline eng/jacoco-test-coverage/pom.xml:152]: Curious - did you use patch script for patching these libraries?

Comment (alzimmermsft) [reply eng/jacoco-test-coverage/pom.xml:152]: I didn't, does the patch script work for releases out of main?

**Review by srnagar — APPROVED** [review]

**Pull request merged**